arangod/Cluster/v8-cluster.cpp

```diff
@@ -1403,15 +1403,16 @@ static void PrepareClusterCommRequest(v8::FunctionCallbackInfo<v8::Value> const&
     }
   }
 
-  coordTransactionID = 0;
+  coordTransactionID = TransactionId::none();
```

Comment:
Using transaction ids here is really misleading. IMHO this is not a _transaction_, but some request identifier.
Can we convert this to a simple `uint64_t` here, so we don't confuse it with proper transactions?